src/backend/marsha/settings.py

```diff
@@ -679,6 +680,12 @@ def SIMPLE_JWT(self):
                 "marsha.core.simple_jwt.tokens.ResourceAccessToken",
                 "marsha.core.simple_jwt.tokens.UserAccessToken",
             ),
+            # Settings for authentication API
+            "UPDATE_LAST_LOGIN": True,
+            "TOKEN_OBTAIN_SERIALIZER": "marsha.account.serializers.UserTokenObtainPairSerializer",
+            "ROTATE_REFRESH_TOKENS": True,
+            "BLACKLIST_AFTER_ROTATION": True,
+            "TOKEN_REFRESH_SERIALIZER": "marsha.account.serializers.UserTokenRefreshSerializer",
```

Comment:
Can you also remove the `ACCESS_TOKEN_LIFETIME` settings to use the default one ?